doc/API/centreon-api-v2.yaml

```diff
@@ -3329,8 +3398,7 @@ components:
         - type: object
           properties:
             check_attempt:
-              type: integer
-              format: int64
+              type: string
```

Comment:
are you sure it should be a string ? too lines bellow, the example shows an int

Reply:
couldn't it be nullable too ? like criticallity

Reply:
currently, seems not be nullable.
It should be a int instead of a string. But I made the PR to fix the current documentation. Typing will be updated later